stable/artifactory/templates/artifactory-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.art_name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.art_replicaCount }}
```

Comment:
Don't use underscores. Use nesting instead.

See https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md

Again, a good example would be https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress/.